sky/dist/BUILD.gn

```diff
@@ -24,7 +24,7 @@ copy("sky_shell") {
     deps = [
       "//sky/shell",
     ]
-  } else if (is_ios) {
+  } else if (is_ios || is_mac) {
     sources = [
       "$root_build_dir/Sky.app",
```

Comment:
This should probably be SkyShell.app these days.  SkyShell is the "general purpose runner for debugging".  Othewise individual "sky apps" are built around .skyx files for specific instances.